link-shortener/src/main/java/com/google/cloud/servicebroker/examples/linkshortener/beans/LinkInfo.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.servicebroker.examples.linkshortener.beans;
+
+import com.google.cloud.servicebroker.examples.linkshortener.enums.ThreatStatus;
+
+public class LinkInfo {
```

Comment:
Consider making this immutable as well. (remove setters and make constructor a single all-arg, null checked)

Reply:
I'm not a huge fan of immutability for POJOs/data objects, because it makes them more difficult to refactor, but I definitely like it for configuration objects where you don't want a client changing something you might be using internally. Even then I'd prefer a builder pattern over an all-arg constructor for readability/maintainability.